test/e2e-go/cli/goal/cicd_expect/goalAlgonetStatusTest.exp

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/expect -f
+set err 0
+log_user 1
+
+if { [catch {
+
+    set ::GLOBAL_TEST_ALGO_DIR ""
+    set ::GLOBAL_TEST_ROOT_DIR ""
+    set ::GLOBAL_NETWORK_NAME ""
+
+    source  ../expect/goalExpectCommon.exp
+    set TEST_ALGO_DIR [lindex $argv 0]
```

Comment:
Do we need to add testing for valid input argument values ?

Reply:
Added check to make sure that the provided directories exist.